.gitignore

```diff
@@ -112,6 +112,8 @@ deployment/ansible/*.retry
 *.aux.xml
 
 /data/
-/notebooks/
 /.vscode/
 /.history/
+
+/notebooks/*
+!/notebooks/**/*.ipynb
```

Comment:
In my PR I would like to modify some parts of the notebooks to use the `Visualizer` class (when appropriate) instead of plotting stuff from scratch. As part of this I will modify the `Visualizer` to work when you just have images and no labels.

Reply:
Do you mean the notebooks in this PR? I think it makes sense for `sampling_training_data.ipynb` because it uses `pytorch_learner` but not the others since they are just using `core` and `pipeline`.

Reply:
Yes, I will just make changes to `sampling_training_data` then.